lib/bacterias.ts

```diff
@@ -6,3 +6,49 @@ export const RESOURCE_TYPES = [
   "Glimmerweave",
   "CelestialOre",
 ];
+
+export type BacteriaDTO = {
+  bacteriaType: number;
+  modifierData: {
+    type: number;
+    description: string;
+  }[];
+  resourcesIncome: {
+    type: number;
+    name: string;
+    amount: number;
+    allTimeAmount: number;
+  }[];
+};
```

Comment:
(just curious) why _"Bacteria"_ 🦠

Reply:
Good question, the naming is extracted from the game files. Lavapotion called it bacteria, and so do I ;).